src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+    private boolean result = false;
+
     public boolean login(String email, String password) {
-        return false;
+        User myUser = userService.findByEmail(email);
+        if (myUser != null && password.equals(myUser.getPassword())) {
+            result = true;
+        } else {
+            result = false;
+        }
+        return result;
```

Comment:
(myUser != null && password.equals(myUser.getPassword()) is an expression that returns a boolean value, and you use it in `if` to return .. a boolean value. In cases like this, we can return the value directly, there's no need for `if`